### PR TITLE
 [REVIEW] Dataset binary and frequency features

### PR DESCRIPTION
**Pull request by brhodes10**
* Added utility in clx to generate binary and frequency features for a given dataframe with specified entity and feature col.

### CODE REVIEW

File: python/clx/features.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            )
+        )
+    df_grouped = (
+        dataframe.groupby([entity_id, feature_id])
+        .count()
+        .reset_index()
+        .set_index(entity_id)
+    )
+    # https://github.com/rapidsai/cudf/issues/1214
+    pdf_grouped = df_grouped.to_pandas()
```
Comment (BartleyR) [inline python/clx/features.py:63]: It looks like you're doing the aggregations separately? If that's the case, could you use `pivot` to do this? https://github.com/rapidsai/cudf/pull/5779

**Review by BartleyR — APPROVED** [review]
Approved, but curious if you could use `pivot` to accomplish this.

File: python/clx/features.py (hunk 1 above)
Comment (brhodes10) [reply python/clx/features.py:63]: Sure I can give `pivot` a try. Thanks!

**Review by tzemicheal — APPROVED** [review]

**Review by raykallen — APPROVED** [review]

**Review by raydouglass — APPROVED** [review]

**Review by efajardo-nv — APPROVED** [review]

**Comment by efajardo-nv** [comment]
rerun tests

**Comment by efajardo-nv** [comment]
rerun tests

**Comment by efajardo-nv** [comment]
PR build errors are coming from #358.

**Comment by brhodes10** [comment]
@gpucibot merge

**Pull request merged**